internal/cdi/cdi.go

```diff
@@ -66,6 +69,18 @@ func newHandler(opts ...Option) (Interface, error) {
 	if !c.deviceListStrategies.IsCDIEnabled() {
 		return &null{}, nil
 	}
+	if c.driverRoot == "" {
```

Comment:
Yes, that's probably better to do, but is redundant since we 1. set it in the caller, and 2. also set it in the CDI api. Let me clean it up regardless.